server/src/controllers/Auth.controller.js

```diff
@@ -8,7 +8,7 @@ import encrypt from '../helper/encrypt';
 dotenv.config();
 
 const { users } = dummyData;
-const { encryptPassword } = encrypt;
+const { encryptPassword, comparePassword } = encrypt;
```

Comment:
Is it not possible to import these directly on line 6 as in 
```suggestion
import { encryptPassword, comparePassword } from '../helper/encrypt';;
```
Also for line 10 i.e ```import { users } from '../dummyDb/db';```

Reply:
I have this blocker, it has to do with eslint issue "import/named ".... I have been on it for hours please help me out